app/Services/Model/Imp/SponsorUserInfoGrantService.php

```diff
@@ -146,8 +148,19 @@ public function addBadgeScan(Summit $summit, Member $current_member, array $data
             $scan->setUser($current_member);
             $scan->setBadge($badge);
             $scan->setNotes(isset($data['notes'])? trim($data['notes']): "");
+
             $sponsor->addUserInfoGrant($scan);
 
+            // extra questions
+            $extra_questions = $data['extra_questions'] ?? [];
+
+            if (count($extra_questions)) {
```

Comment:
what is the rationale behind this conditional validation
questions should be validated always